mad_spark_multiagent/run_madspark.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Run MadSpark with environment variables loaded
+
+# Activate virtual environment
+source venv/bin/activate
+
+# Load environment variables from .env file
+# set -a enables automatic export of all variables
+# set +a disables it after sourcing
+set -a
+source .env
```

Shellcheck (0.10.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)